components/Account/Account.jsx

```diff
@@ -0,0 +1,118 @@
+import { useState, useEffect } from "react"
+import { useUser, useSupabaseClient } from "@supabase/auth-helpers-react"
+import styles from "./Account.module.css"
+import cn from "classnames"
+import Avatar from "@/components/Avatar/Avatar"
+
+export default function Account({ session }) {
+  const supabase = useSupabaseClient()
+  const user = useUser()
+  const [loading, setLoading] = useState(true)
+  const [name, setName] = useState(null)
+  const [avatarUrl, setAvatarUrl] = useState(null)
+
+  useEffect(() => {
+    async function getProfile() {
+      try {
+        setLoading(true)
+
+        let { data, error, status } = await supabase
+          .from("profiles")
+          .select(`*`)
+          .eq("id", user.id)
+          .single()
+
+        if (error && status !== 406) {
+          throw error
+        }
+
+        if (data) {
+          setName(data.name)
+          setAvatarUrl(data.avatar)
```

Comment:
It would be worth updating these two column names:
`name` => `username` — `name` can potentially get confused with other parts of the app
`avatar` => `avatar_url` — better reflects what's getting passed into this field